tests/Xamarin.Android.Tools.Bytecode-Tests/KotlinMetadataTests.cs

```diff
@@ -181,9 +181,7 @@ public void Object ()
 			var klass_meta = GetClassMetadata ("Object.class");
 
 			Assert.AreEqual (KotlinClassInheritability.Final, klass_meta.Inheritability);
-
-			// One would expect this to be "Object" but it doesn't seem to be
-			Assert.AreEqual (KotlinClassType.Class, klass_meta.ObjectType);
+			Assert.AreEqual (KotlinClassType.Object, klass_meta.ObjectType);
```

Comment:
This change seems weird; how'd it carries from the `.csproj` migration?

Reply:
Apparently I had committed a bad `.class` file previously.  It got regenerated from the Kotlin as part of testing this, which now gave me the results I was previously expecting.